src/models/ledger/ledger.ts

```diff
@@ -16,5 +17,5 @@ export interface Ledger {
   parent_hash: string;
   total_coins: string;
   transaction_hash: string;
-  transactions?: Transaction[];
+  transactions?: Array<Transaction & { metaData?: Metadata }>;
```

Comment:
This should probably import from src/models/metadata.ts now to get TransactionMetaData (Merged as part of #1572 )